genesyscloud/outbound_ruleset/resource_genesyscloud_outbound_ruleset.go

```diff
@@ -56,7 +56,7 @@ var (
 			},
 			`actions`: {
 				Description: `The list of actions to be taken if the conditions are true.`,
-				Optional:    true,
+				Required:    true,
```

Comment:
Although the api accepts rulesets without actions it leads to a bug, the outbound team is fixing the api to make it required but I made it required here to avoid issues now